tajo-core/src/main/java/org/apache/tajo/master/rm/Worker.java

```diff
@@ -34,17 +39,37 @@
   /** class logger */
   private static final Log LOG = LogFactory.getLog(Worker.class);
```

Comment:
The name of this class looks less intuitive. How about changing it to a more intuitive name such as LiveWorkerInfo? If you have a good idea, please talk to me.